lib/samples/style_graphics_with_symbols/style_graphics_with_symbols.dart

```diff
@@ -0,0 +1,322 @@
+
+    // Create the graphic with polyline and symbol.
+    final graphic = Graphic(
+      geometry: polyline,
+      symbol: lineSymbol,
+    );
+
+    // Add graphic to the graphics overlay.
+    _graphicsOverlay.graphics.add(graphic);
+  }
+
+  void _createPolygon() {
+    // Create a green dash line symbol
+    final outlineSymbol = SimpleLineSymbol(
+      style: SimpleLineSymbolStyle.dash,
+      color: Colors.green,
+      width: 1,
+    );
+
+    // Create a green mesh simple fill symbol.
+    final fillSymbol = SimpleFillSymbol(
+      style: SimpleFillSymbolStyle.diagonalCross,
+      color: Colors.green,
+      outline: outlineSymbol,
+    );
+
+    // Create a new point collection for polygon.
+    final polygonBuilder =
+        PolygonBuilder(spatialReference: SpatialReference.wgs84);
+    polygonBuilder.addPointXY(
+      x: -2.6425,
+      y: 56.0784,
+    );
+    polygonBuilder.addPointXY(
+      x: -2.6430,
+      y: 56.0763,
+    );
+    polygonBuilder.addPointXY(
+      x: -2.6410,
+      y: 56.0759,
+    );
+    polygonBuilder.addPointXY(
+      x: -2.6380,
+      y: 56.0765,
+    );
+    polygonBuilder.addPointXY(
+      x: -2.6380,
+      y: 56.0784,
+    );
+    polygonBuilder.addPointXY(
+      x: -2.6410,
+      y: 56.0786,
+    );
+    final polygon = polygonBuilder.toGeometry();
+
+    // Create the graphic with polygon and symbol.
+    final graphic = Graphic(geometry: polygon, symbol: fillSymbol);
+
+    // Add graphic to the graphics overlay.
+    _graphicsOverlay.graphics.add(graphic);
+  }
+
+  void _createText() {
+    // Create two text symbols.
+    final bassRockTextSymbol = TextSymbol(
+      text: 'Black Rock',
+      color: Colors.blue,
+      size: 10,
+      horizontalAlignment: HorizontalAlignment.left,
+      verticalAlignment: VerticalAlignment.bottom,
+    );
+
+    final craigleithTextSymbol = TextSymbol(
+      text: 'Craigleith',
+      color: Colors.blue,
+      size: 10,
+      horizontalAlignment: HorizontalAlignment.right,
+      verticalAlignment: VerticalAlignment.top,
+    );
+
+    // Create two points.
+    final bassPoint = ArcGISPoint(
+      x: -2.64,
+      y: 56.079,
+      spatialReference: SpatialReference.wgs84,
+    );
+    final craigleithPoint = ArcGISPoint(
+      x: -2.72,
+      y: 56.076,
+      spatialReference: SpatialReference.wgs84,
+    );
+
+    // Create two graphics from the points and symbols.
+    final bassRockGraphic = Graphic(
+      geometry: bassPoint,
+      symbol: bassRockTextSymbol,
+    );
+    final craigleithGraphic = Graphic(
+      geometry: craigleithPoint,
+      symbol: craigleithTextSymbol,
+    );
+
+    // Add graphics to the graphics overlay.
+    _graphicsOverlay.graphics.addAll([
+      bassRockGraphic,
+      craigleithGraphic,
+    ]);
+  }
+
+  void _setExtent() async {
+    // Get all of the graphics contained in the graphics overlay.
+    List<Graphic> myGraphicCollection = _graphicsOverlay.graphics;
```

Comment:
This could just be `final myGraphicCollection = _graphicsOverlay.graphics;`

Or you don't even really need `myGraphicCollection`, you could just use `_graphicsOverlay.graphics` directly in the for loop below, like `for (final graphic in _graphicsOverlay.graphics) {`